tests/test_public_api_repr.py

```diff
@@ -230,7 +230,6 @@ def test_utils_public_api():
         "digest",
         "load_schema",
         "options_from_eqdelimstring",
-        "remove_prefix",
```

Comment:
To make sure we get all these deprecations in the release notes properly I should start a release notes for `v0.6.2` and then make sure it gets added to PR #1438 (or if that gets merged first that it gets added to them) and then have this noted there too.